pkg/controller/admissionchecks/multikueue/indexer_test.go

```diff
@@ -38,7 +38,7 @@ const (
 	TestNamespace = "ns"
 )
 
-func getClientBuilder() (*fake.ClientBuilder, context.Context) {
+func getClientBuilder(t *testing.T) (*fake.ClientBuilder, context.Context) {
```

Comment:
```suggestion
func getClientBuilder(ctx context.Context) *fake.ClientBuilder {
```
Maybe like this? WDYT?